### PR TITLE
chore: extract tab function and add tests

### PR DESCRIPTION
**Pull request by deboer-tim**
### What does this PR do?

From a post-merge comment on PR #3227: tabbing should be extracted to a shared/common function, and tests should be added.

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

N/A

### How to test this PR?

N/A, automated testing should be sufficient.

### CODE REVIEW

File: packages/renderer/src/lib/dialogs/dialog-utils.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+export class DialogUtils {
```
Comment (benoitf) [inline packages/renderer/src/lib/dialogs/dialog-utils.ts]: should we have a test for tabWithinParent ?

Comment (deboer-tim) [reply packages/renderer/src/lib/dialogs/dialog-utils.ts]: Added tabbing util test.

**Review by lstocchi — APPROVED** [review]
LGTM

**Comment by deboer-tim** [comment]
Rebased on main.

File: packages/renderer/src/lib/dialogs/dialog-utils.ts (hunk 1 above)
Comment (benoitf) [inline packages/renderer/src/lib/dialogs/dialog-utils.ts]: I'm wondering why you didn't directly exported a function

```
export function tabWithinParent(...
```

as the class is not required here as you don't want to instantiate anyway. Probably a Java design.

Comment (deboer-tim) [reply packages/renderer/src/lib/dialogs/dialog-utils.ts]: lol, yes, old habits die hard. New commit fixes this.

Comment (benoitf) [reply packages/renderer/src/lib/dialogs/dialog-utils.ts]: it was just a remark. thanks for the refactoring 👍

**Review by benoitf — APPROVED** [review]
thanks for the refactoring and the tests !

**Pull request merged**